### DISCUSSION TITLE
TypeError: f.some is not a function

### DISCUSSION THREAD

**Discussion by amishaaggarwal** [tailwindlabs/headlessui#3142] [Help]
![Screenshot from 2024-04-26 14-36-17](https://github.com/tailwindlabs/headlessui/assets/67583190/ae5c9a1c-1b21-4060-aa37-858cddf3c4cc)
this error occurs when clicked on combobox dropdown with multiple selection enabled

**Comment by RobinMalfait** [comment]
Hey!

You probably have the multiple prop enabled but the value you provided is probably not an array.

can you share a minimal reproduction repo?
